sklearn/svm/_classes.py

```diff
@@ -625,6 +641,14 @@ def __init__(self, C=1.0, kernel='rbf', degree=3, gamma='scale',
             break_ties=break_ties,
             random_state=random_state)
 
+    def _more_tags(self):
+        return {
+            '_xfail_test': {
+                'check_sample_weights_invariance(kind=zeros)':
+                'zero sample_weight is not equivalent to removing samples',
+            }
+        }
```

Comment:
Of course the amount of repetitions could be reduced by defining this tag in `BaseSVC` but the expectations is that these estimators would be fixed one by one and it's easier to understand what needs fixing this way.